src/show/pages/public/Explore/index.js

```diff
@@ -0,0 +1,89 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Button, ProgressBar } from '../../../components';
+
+const Explore = () => {
+  const [projectCard, setProjectCard] = useState([]);
+  const [selectedCategory, setSelectedCategory] = useState('');
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    axios
+      .get(`http://localhost:3000/v1/projects?category=${selectedCategory}`)
+      .then(res => setProjectCard(res.data))
+      .catch(err => err);
+  }, [selectedCategory]);
+
+  const categories = [
+    'garden',
+    'house',
+    'tech',
+    'cars',
+    'office',
+    'audio',
+    'backpacks',
+    'clothes'
+  ];
+
+  return (
+    <div className='h-full w-full mt-10'>
+      <div className='flex justify-center items-center bg-white h-12'>
+        {categories.map((category, index) => (
+          <div
+            onClick={() =>
+              selectedCategory === category
+                ? setSelectedCategory('')
+                : setSelectedCategory(category)
+            }
+            className={
+              selectedCategory === category
+                ? 'bg-primary-50 text-white mx-1 rounded-full px-2 transition-all duration-300 cursor-pointer'
+                : 'border-2 border-primary-50 rounded-full mx-1 px-2 text-primary-50 transition-all duration-300 cursor-pointer'
+            }
```

Comment:
There are classes that repeat themselves here. So you'll have to concat the conditional with the classes  that are for both, the true and false branch. Also put this big conditional into a constant.

Reply:
done